components/admission-webhook/Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV GO111MODULE=on
 RUN go build  -o webhook .
 
 # Copy the controller-manager into a thin image
-FROM ubuntu:latest
+FROM gcr.io/distroless/base:latest
```

Comment:
Do you want to use base-debian10? 
see
https://github.com/GoogleContainerTools/distroless#base-operating-system

Do you need base or could you get by with just static?
https://github.com/GoogleContainerTools/distroless/blob/master/base/README.md

Reply:
https://github.com/kubeflow/kubeflow/pull/4775/files#r382505839